docs/configuration.md

```diff
@@ -0,0 +1,115 @@
+# Configuring and Tuning Cluster Logging
+
+OKD cluster logging is configurable by modifying the CustomResource deployed
+in the *openshift-logging* namespace.  The [sample](../hack/cr.yaml) resource
+defines a complete cluster logging deployment which includes all the subcomponents
+of the logging stack to collect, store and visualize logs.  The `cluster-logging-operator`
+watches the CustomResources and adjusts the logging deployment accordingly.
+
+## Management State
+Cluster logging is `Managed` by the `cluster-logging-operator`.  It can be placed
```

Comment:
Do we need markup for Managed? 
Cluster mogging is managed by the `cluster-logging-operator`.